stream.go

```diff
@@ -0,0 +1,145 @@
+package tarantool
+
+import (
+	"fmt"
+	"time"
+
+	"gopkg.in/vmihailenco/msgpack.v2"
+)
+
+const (
+	// By default, the isolation level of Tarantool is serializable.
+	defaultIsolationLevel = iota
+	// The ReadCommittedLevel isolation level makes visible all transactions
+	// that started commit (stream.Do(NewCommitRequest()) was called).
+	ReadCommittedLevel
+	// The ReadConfirmedLevel isolation level makes visible all transactions
+	// that finished the commit (stream.Do(NewCommitRequest()) was returned).
+	ReadConfirmedLevel
+	// If the BestEffortLevel (serializable) isolation level becomes unreachable,
+	// the transaction is marked as «conflicted» and can no longer be committed.
+	BestEffortLevel
+)
+
+type Stream struct {
+	Id   uint64
+	Conn *Connection
+}
+
+func fillBegin(enc *msgpack.Encoder, txnIsolation uint64, timeout time.Duration) error {
```

Comment:
A separate type will look good here (alias for uint64).
```suggestion
func fillBegin(enc *msgpack.Encoder, txnIsolation TxnIsolation, timeout time.Duration) error {
```

Reply:
Alias added:

https://github.com/tarantool/go-tarantool/blob/f456b45d0277c878ed577253f844db850ee741f5/stream.go#L10